.github/workflows/deprecate_version.yml

```diff
@@ -0,0 +1,31 @@
+name: Deprecate X-Ray Node SDK Version
+on:
+  workflow_dispatch:
+
+jobs:
+  deprecate_xray_node_sdk_version:
+    name: Deprecate X-Ray Node SDK version in NPM registry
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - run: npm install -g npm@8.19.4
+
+      - name: Deprecate Version 3.7.0
+        run: |
+          npm deprecate aws-xray-sdk@3.7.0 "3.7.0 is deprecated due to known issue in Lambda"
```

Comment:
Is there an issue that we can link to in the deprecation message for further clarity?

Reply:
I see a couple here: 
https://github.com/aws/aws-xray-sdk-node/issues/654
https://github.com/aws/aws-xray-sdk-node/issues/655

Reply:
Are these not why we are deprecating 3.7.0?

Reply:
We found the issue locally first, but you are right, these are the same issue.